README.md

```diff
@@ -9,6 +9,10 @@ provided to fetch all these values.
 up to date, however this package may lag behind. If this is impacting your development,
 please file an [issue](https://github.com/strava/go.strava/issues) and it will be addressed as soon as possible.
 
+#### Starting June 2018
+
+Find the new Strava V3 API at https://github.com/strava/developers.strava.com/, you can generate Go client following the [Client code section](https://github.com/strava/developers.strava.com/#client-code). This repository is no longer getting updated and may get removed in the near future.
```

Comment:
nit: `...at https://github.com/strava/developers.strava.com/. You can...`

other nit: `...may be removed...`